capa/main.py

```diff
@@ -780,6 +807,28 @@ def get_extractor_from_cli(args, input_format: str, backend: str) -> FeatureExtr
         raise ShouldExitError(E_CORRUPT_FILE) from e
 
 
+def get_extractor_filters_from_cli(args, input_format) -> Dict[str, Set]:
+    if input_format in STATIC_FORMATS:
```

Comment:
we may want to factor out the `is_static_analysis` into its own routine. though its not required in this PR.